Runtime/Scripts/DebugUtils/RGGizmosWrapper.cs

```diff
@@ -0,0 +1,383 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using RegressionGames.RGBotConfigs;
+using UnityEngine;
+using UnityEditor;
+using Object = UnityEngine.Object;
+
+namespace RegressionGames.DebugUtils
+{
+    
+    /**
+     * A set of debug utilities for drawing gizmos and text on top of entities in your scene.
+     */
+    // ReSharper disable once InconsistentNaming
+    public class RGGizmos
+    {
+        
+        private readonly Dictionary<string, Tuple<int, Vector3, Color>> _linesFromEntityToPosition = new();
+        private readonly Dictionary<string, Tuple<int, int, Color>> _linesFromEntityToEntity = new();
+        private readonly Dictionary<string, Tuple<Vector3, Vector3, Color>> _linesFromPositionToPosition = new();
+        private readonly Dictionary<string, Tuple<Vector3, Color, float, bool>> _spheresAtPosition = new();
+        private readonly Dictionary<string, Tuple<int, Color, float, bool>> _spheresAtEntity = new();
+        private readonly Dictionary<int, Tuple<string, float>> _billboardsToDraw = new();
+        private readonly Dictionary<int, GameObject> _drawnBillboards = new();
+        
+        // Billboard text objects
+        private readonly GameObject _billboardAsset;
+
+        public RGGizmos()
+        {
+            var path = "Packages/gg.regression.unity.bots/Runtime/Prefabs/AgentBillboardText.prefab";
+            _billboardAsset = AssetDatabase.LoadAssetAtPath<GameObject>(path);
```

Comment:
@rcornwall-reg you mention that there are better ways of doing this... after much research and testing, I still couldn't figure it out. Have any ideas/references/want to hop on a call today to help walk me through it?

Reply:
It's not a _bad_ way of doing it, and it should work, I just think directly referencing prefabs by their file path is bad practice for Unity development, Here are some alternatives, but of course feel free to disregard

1. I would recommend the `Resource.Load` method. It doesn't require hardcoding the path to the prefab, but it does require that the prefab be located in a Resources folder, and you’d refer to it by its name.
 * Place prefab into Resources folder: `Resources/AgentBillboardText.prefab`
 * Load prefab with `_billboardAsset = Resources.Load<GameObject>("AgentBillboardText");`
 * Resources.Load will still work even if Resources folder is in a package
 
 Other than that you could:
 2. Expose the prefab as a field (either public or mark with `[SerializeField]`, although it'd require linking it in the inspector so it's not my favorite
 
 3. Including prefab as part of a 'RGGizmosSettings` scriptable object that is added to the wrapper - again probably as an inspector field. I'd recommend this method if there's going to be a lot of prefabs referenced or other settings values
 
Any of these would work and make the prefab loading a lot less prone to error

Reply:
I will try the Resource approach! From my reading online... it seemed like packages actually don't support the resource folder, but if that works, that would make things easier!

Reply:
Thanks for this! Got it to work with Resource.Load and seems cleaner, thanks!